docs/get-started/deployment-and-usage.md

```diff
@@ -133,16 +133,17 @@ scenarios under
 
 #### Master-worker pod
 
-You can also run nfd-master and nfd-worker inside the same pod
+You can also run nfd-master, nfd-worker and nfd-topology-updater inside the same pod
```

Comment:
This isn't true, topology-updater is not deployed by the `default-combined` overlay. You could mention that. And possibly add that in order to enable topology-updater in this scenario the user must customize the deployment herself.